pkg/policyreport/builder.go

```diff
@@ -254,23 +257,54 @@ func buildViolatedRules(er *response.EngineResponse) []kyverno.ViolatedRule {
 }
 
 const categoryLabel string = "policies.kyverno.io/category"
+const severityLabel string = "policies.kyverno.io/severity"
+
+type annotationValues struct {
+	category string
+	severity report.PolicySeverity
+}
+
+func (av *annotationValues) setSeverityFromString(severity string) {
+	switch severity {
+	case report.SeverityHigh:
+		av.severity = report.SeverityHigh
+	case report.SeverityMedium:
+		av.severity = report.SeverityMedium
+	case report.SeverityLow:
+		av.severity = report.SeverityLow
+	}
+}
+
+func (builder *requestBuilder) fetchAnnoationValues(policy, ns string) annotationValues {
+	av := annotationValues{}
+	ann := builder.fetchAnnoations(policy, ns)
+
+	if category, ok := ann[categoryLabel]; ok {
+		av.category = category
+	}
+	if severity, ok := ann[severityLabel]; ok {
+		av.setSeverityFromString(severity)
+	}
+
+	return av
+}
 
-func (builder *requestBuilder) fetchCategory(policy, ns string) string {
+func (builder *requestBuilder) fetchAnnoations(policy, ns string) map[string]string {
```

Comment:
Shouldn't this be `fetchAnnotations` instead? @realshuting @fjogeleit

Reply:
you're right. I will fix this.